book/docs/Documentation.ipynb

```diff
@@ -0,0 +1,60 @@
+{
```

Comment:
<p>exegesis &lt;- I don't think this is the right word here, too academic and also seems to <code>text</code> specifically</p><p> </p><p>Consider breaking some of the paragraph lists into bullet lists for better web readbility, and lessen the density of letters (better whitespace ratio)</p><p> </p><p>Github - be careful about suggesting people put Data in their git repositories, this is not a best practice. Some frameworks like <a href="https://osf.io/" target="_blank">OSF</a> exist to help tie together large data and code - hosted in github)</p><p> </p><p>JupyterBook - Common alternatives (Quarto which also handles R)</p><p>Documentation can exist in the same repo as code, often under a docs folder, and on github can be autobuilt into a website for the repo. I'd say this is actually the most common on projects. A separate documentation repo is more common on big projects with lots of commits and tons of code.</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/developmentseed/scaling_science/pull/9/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
Copy that on wording and spacing. 

Oops yes didn't meant to include "data" in the github section. Meant to break that out into a separate platform.

Noted on Quarto. Will mention.

Will update on the Documentation hosting. I meant to suggest that the actually viewing of the docs often lives outside of the Github interface itself, whether that's a website built from HTML built on top of markdown etc.